Try_Catch.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream>
+#include <stdexcept>
+
+using namespace std;
+
+class CarDamageException : public exception {
+private:
+    string message;
+public:
+    CarDamageException(const char* msg) : message(msg) {}
+    const char* what() const noexcept override {
+        return message.c_str();
+    }
+};
+
+class BrakeFailureException : public exception {
+private:
+    string message;
+public:
+    BrakeFailureException(const char* msg) : message(msg) {}
+    const char* what() const noexcept override {
+        return message.c_str();
+    }
+};
+
+class Car {
+public:
+    void driveSafely(bool collision, bool brakeFailure) {

```

Comment:
what if there are more exceptions/reasons for car accidents, then it wouldn't be the correct approach to pass the boolean values through parameters. can think of a better approach where we can handle multiple exceptions.

Reply:
Done